src/test/calculate.test.js

```diff
@@ -0,0 +1,8 @@
+import calculate from '../logic/calculate';
+
```

Comment:
- [ ] Great job with the test, they are all ok. please kindly add more test to test the scenarios of the calculate.js module. u added only for returning an object. check scenarios like User pressed an operation button and there is an existing operation, If there is no operation, update next and clear the value. great job 👍🏾